backend/config.js

```diff
@@ -29,6 +29,22 @@ module.exports = {
     port: process.env.REDIS_PORT || 6379,
     password: process.env.REDIS_PASSWORD,
   },
+  applications: {
```

Comment:
Created this because didn't feel right putting them within`apis`.  Think some of the others should be moved here too at a later date, `omic` and `bookVideoLink` for example.  Happy to discuss if you think otherwise though.